tests/transports-integrations/tests/utils/common.py

```
@@ -0,0 +1,416 @@
+def mock_tool_response(tool_name: str, args: Dict[str, Any]) -> str:
+    """Generate mock responses for tool calls"""
+    if tool_name == "get_weather":
+        location = args.get("location", "Unknown")
+        unit = args.get("unit", "fahrenheit")
+        return f"The weather in {location} is 72°{'F' if unit == 'fahrenheit' else 'C'} and sunny."
+
+    elif tool_name == "calculate":
+        expression = args.get("expression", "")
+        try:
+            # Simple evaluation for basic math
+            result = eval(expression.replace("x", "*").replace("×", "*"))
```

Ruff (0.11.9)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)